common/djangoapps/student/views.py

```diff
@@ -701,6 +701,10 @@ def dashboard(request):
         redirect_message = _("The course you are looking for does not start until {date}.").format(
             date=request.GET['notlive']
         )
+    elif 'course_closed' in request.GET:
+        redirect_message = _("The course you are looking for is closed for enrollment as of {date}.").format(
```

Comment:
@srpearce, this message will be shown to the user at the top of their dashboard if they try to go to the track selection page for a course which is closed for enrollment. Does that sound good?